### PR TITLE
Add --follow-shortcuts to ls Command

### PR DESCRIPTION
**Pull request by nrasulicpfm**
This pull request adds support for shortcuts when using the `ls` command.
The command would add `--follow-shortcuts` flag similar to `upload` and `download`

### CODE REVIEW

File: solvebio/cli/data.py [hunk 1]
```diff
@@ -942,7 +942,9 @@ def ls(args):
               "Try the --recursive flag instead.")
         return False
 
-    files = _ls(args.full_path, recursive=args.recursive)
+    follow_shortcuts = True if args.follow_shortcuts else False
```
Comment (davecap) [inline solvebio/cli/data.py:945]: why do we need this if we can just use `args.follow_shortcuts`

File: solvebio/cli/data.py [hunk 2]
```diff
@@ -954,17 +956,38 @@ def ls(args):
...
+                    "{}  {}  {}  from shortcut: {}".format(
+                        resolved_file.last_modified,
+                        resolved_file.object_type.ljust(8),
+                        resolved_file.full_path.ljust(50),
+                        shortcut
+                    )
+                )
+            except NotFoundError:
+                print(
+                    "shortcut target not found!{}from shortcut: {}".format(
```
Comment (davecap) [inline solvebio/cli/data.py:977]: can we rephrase it to indicate that it may not be accessible to the current user?
`Shortcut {} could not be resolved: the target may have been deleted or you may not have permission to access it`

**Review by davecap — CHANGES_REQUESTED** [review]
Minor changes requested

File: solvebio/cli/data.py (hunk 2 above)
Comment (nrasulicpfm) [reply solvebio/cli/data.py]: Addressed these comments a while ago - forgot to follow up here in the thread sorry

File: solvebio/cli/data.py (hunk 1 above)
Comment (nrasulicpfm) [reply solvebio/cli/data.py]: Addressed these comments a while ago - forgot to follow up here in the thread sorry

**Review by davecap — APPROVED** [review]

**Comment by davecap** [comment]
@nrasulicpfm can we merge this?
